modules/stochastic_tools/test/tests/reporters/directperturbation/tests

```diff
@@ -0,0 +1,46 @@
+[Tests]
+  design = DirectPerturbationReporter.md
+  issues = '#28671'
+
+  [central_difference]
+    requirement = 'The system shall support the ability to compute direct perturbation-based sensitivity values with central differencing '
+    [distributed]
+      type = JSONDiff
+      input = dp_main.i
+      jsondiff = dp_main_out.json
+      allow_test_objects = true
+      detail = 'on distributed data sets;'
+      min_parallel = 4
+      max_parallel = 4
```

Comment:
Why those restrictions?

Reply:
Aaaa, right! I forgot that we already have a test with `-p 2`. That should actually be enough for this. Removing this.